tests/test_type_utils.py

```diff
@@ -150,3 +150,46 @@ def test_validate_types_sad(type_):
 )
 def test__safe_subclass(candidate, type_, expected):
     assert htypes._safe_subclass(candidate, type_) == expected
+
+
+@pytest.mark.parametrize(
```

Comment:
This should assert the output if its easy...

Reply:
The issue I fear is that the output depends on the Python version (might not be the case). I currently don't have the full CI/CD system locally to test on all Python versions efficiently and wanted to push a fix sooner than later

Reply:
Fair enough, let's leave it at this 👍